Examples/trame/app.py

```diff
@@ -0,0 +1,305 @@
+import os
+import tempfile
+
+from trame import state
+from trame.layouts import SinglePageWithDrawer
+from trame.html import vtk, vuetify
+
+from vtkmodules.vtkFiltersSources import vtkConeSource
+from vtkmodules.vtkRenderingCore import (
+    vtkActor,
+    vtkPolyDataMapper,
+    vtkRenderer,
+    vtkRenderWindow,
+    vtkRenderWindowInteractor,
+)
```

Comment:
since you import vtk as vtkpackage you might want to simply use it rather than using vtkmodules?

Reply:
Thanks for the feedback, we will revisit.

### Background

Since I initially thought I add to discover the relevant modules associated with each classes, in the interest of time it was easier to import from `vtkpackage`, especially considering I was adapting a script written by @aronhelser.